napari/components/text_overlay.py

```diff
@@ -25,7 +31,16 @@ class TextOverlay(EventedModel):
 
     # fields
     visible: bool = False
-    color: str = "#FFFFFF"
+    color: Optional[Array[float, (4,)]] = (1.0, 1.0, 1.0, 1.0)
     font_size: int = 10
     position: TextOverlayPosition = TextOverlayPosition.TOP_LEFT
     text: str = ""
+
+    @validator('color', pre=True)
+    def _coerce_color(cls, v):
+        if v is None:
+            return v
+        elif len(v) == 0:
+            return None
+        else:
```

Comment:
See comment above, I don't think we need the return `None` here. I know that was in the code I copied over from the ColorManager, but I think that was to deal with `current_color` when nothing was selected. Here there will always be text that always needs a color. If I've gotten this wrong maybe @kevinyamauchi can weigh in

Reply:
Sorry for the confusion, @lukasz-migas ! @sofroniewn was right - the `None` that was returned from those color validators was to deal with the case when a color isn't provided.

Reply:
@kevinyamauchi Thanks for clarification, I've removed the if/else from the validator since we always want color.